peer-signer/README.md

```diff
@@ -0,0 +1,26 @@
+# Peer Signer
```

Comment:
I have some problem with telling which part is more important here: signing the secret message (and then verifying is just an additional feature for ensuring that everything is ok) or the verifying setup, where signing is the chosen way. 

Inferring from the package name, former seems to be the crucial thing... but anyway this is a tad confusing to have this kind of structure that the whole package is named after one of its crates, while the other apparently seem to be of equal importance.

I would either rename the package or be more explanatory in README (my preference) / Cargo.toml description.

Reply:
Yes, the split is even deeper: the signer is intended to be shipped as a part of the "setup package" in the `aleph-node-runner` repo, meaning it will eventually be exposed in a binary form to the end-users. The verifier, on the other hand, is meant for internal use by the Aleph team. I think it could be worth mentioning.

Reply:
Maybe it could be renamed to `peer-verification`? I know this places a tad more importance on verifier but this is a tool intended for peer verification. Signer part will be used by a validator candidate, and Verifier part will be used by us. I can add more explanation in README regarding a split of this package.

Reply:
For now I updated the README. Right now I'm leaning more towards name `peer-verifier`/`peer-verification`. Please let me know what do you think about those.

Reply:
in that case I would support you in changing the name; `peer-verification` sounds subtly better for me, since `peer-verifier` suggests a single tool, whereas `peer-verification` should be a fair umbrella term